codeforamerica/vita-min
# FYST-327

## [FYST-327](https://codeforamerica.atlassian.net/browse/FYST-327)
## Is PM acceptance required?
- Yes
## What was done?
- If an intake has a stored submission_pdf attachment, we should use this when a client requests their PDF.
## How to test?
- I tested on staging
- Create a new completed AZ intake using the Bert & Ernie example.
- Download the PDF : It should work exactly as before but faster!
![image](https://github.com/user-attachments/assets/9463ed0d-a362-40e5-9e70-b87d86bf9850)

- Manually enter the following in the URL bar: https://staging.fileyourstatetaxes.org/en/questions/name-dob
- Change the name to "Kermit" and save:
![image](https://github.com/user-attachments/assets/14d4ee62-f386-4bac-b62e-4e762a304bdc)

- Go back to the submission confirmation URL: https://staging.fileyourstatetaxes.org/en/questions/submission-confirmation
- Click the download button. You are still Bert! (Your name in the PDF should NOT have changed)
![image](https://github.com/user-attachments/assets/1cd2e075-f5d2-4c43-86c9-30f32b18ff1c)

[FYST-327]: https://codeforamerica.atlassian.net/browse/FYST-327?atlOrigin=eyJpIjoiNWRkNTljNzYxNjVmNDY3MDlhMDU5Y2ZhYzA5YTRkZjUiLCJwIjoiZ2l0aHViLWNvbS1KU1cifQ
